bedrock/firefox/templates/firefox/new/desktop/download.html

```diff
@@ -195,7 +195,7 @@ <h3 class="mzp-u-title-xs">{{ feature3_title }}</h3>
       </ul>
 
     {% if LANG.startswith('en-') %}
-      <p class="c-notes"><a class="mzp-c-cta-link" href="{{ url('firefox.notes') }}" data-cta-text="See Release Notes">See Release Notes</a></p>
+      <p class="c-notes"><a class="mzp-c-cta-link" href="https://www.mozilla.org/{{ LANG }}/firefox/releasenotes/{{ referrals }}" data-cta-text="See Release Notes">See Release Notes</a></p>
```

Comment:
I get that `referrals` are the querstring UTM params, and am assuming they are manually added because `url()` used to automatically do it for us, but I can't see where `url()` used to use them! 🙃

Reply:
UTM params are for external links only, so we would not add them for internal `url()` links